eng/common/cross/toolchain.cmake

```diff
@@ -36,6 +36,9 @@ elseif(TARGET_ARCH_NAME STREQUAL "arm64")
   if("$ENV{__DistroRid}" MATCHES "tizen.*")
```

Comment:
FYI: files under `eng/common` directory are mirrored from https://github.com/dotnet/arcade/tree/main/eng/commo (once or twice a month). To avoid your changes from getting lost in that directory, please open a parallel PR in dotnet/arcade repo.

We usually send a PR  only to dotnet/aracde for non-critical changes under `eng/common` directory, and let the sync bot update in the next round, but it can be done if we send PRs in both repos in parallel (especially if we want to confirm the change is not breaking anything in runtime's CI).

Reply:
> FYI: files under `eng/common` directory are mirrored from https://github.com/dotnet/arcade/tree/main/eng/commo (once or twice a month). To avoid your changes from getting lost in that directory, please open a parallel PR in dotnet/arcade repo.
> 
> We usually send a PR only to dotnet/aracde for non-critical changes under `eng/common` directory, and let the sync bot update in the next round, but it can be done if we send PRs in both repos in parallel (especially if we want to confirm the change is not breaking anything in runtime's CI).

Thanks for review~

Reply:
Also commit to arcade for LoongArch64,
https://github.com/dotnet/arcade/pull/8271